### ISSUE TITLE
Option to disable group animations

### ISSUE THREAD

**Issue by vccrurial** [hyprwm/Hyprland#3551]
### Description

Option to disable animations for tabbed clients (when inserting new windows, moving them etc.)

**Comment by MightyPlaza** [comment]
what animations?

**Comment by vccrurial** [comment]
> what animations?

Sorry if I wasn't clear. I meant the ability to globally enable all animations except for the ones that occur during interactions with groups (adding new clients, closing them etc).

**Comment by MightyPlaza** [comment]
may I ask why?
I believe some of them should be fully removed, not just disableable

**Comment by KorigamiK** [comment]
This would be great actually!

**Comment by sdaqo** [comment]
Yeah, it really does look weird especially when you switch through the clients in the group it animates every switch, that makes it look like the window moves but it actually just stays in place because it is in the group.

**Comment by jheroy** [comment]
The issue occurs when swapping the positions of a group window with other windows. Only the active window's position is actually updated, while all inactive windows remain in their original positions (this can be verified via hyprctl). When switching to a different group window, there is an animation showing the movement from the old position to the current one.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
